src/docker_manager.rs

```diff
@@ -340,18 +340,26 @@ impl DockerNetwork {
             ..Default::default()
         };
 
-        let id = self
+        match self
             .docker
             .create_container::<&str, &str>(Some(create_options), container_config)
             .await
-            .unwrap()
-            .id;
-        self.docker
-            .start_container::<String>(&id, None)
-            .await
-            .unwrap();
-
-        container.id = Some(id.clone());
+        {
+            Ok(container_response) => {
+                let id = container_response.id;
+                match self.docker.start_container::<String>(&id, None).await {
+                    Ok(_) => {
+                        container.id = Some(id.clone());
+                    }
+                    Err(e) => {
+                        panic!("Failed to start container, the ports you are trying to use to run this container might already be used by another process, so try again with new base_ports: {}", e);
```

Comment:
```suggestion
                        panic!("Failed to start the xrpld container, try checking your base port configuration values to make sure they are not bound by another process: {}", e);
```

I think this makes it a bit more clear.